addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -817,9 +817,9 @@
                                     <field name="quantity" optional="hide" attrs="{'readonly': [('product_has_cost', '=', False)]}"/>
                                     <field name="tax_ids" string="Taxes" optional="show" widget="many2many_tags" context="{'default_company_id': company_id}"/>
                                     <field name="amount_tax" optional="hide"  options="{'currency_field': 'currency_id'}" context="{'default_company_id': company_id}" readonly="1" groups="base.group_multi_currency"/>
-                                    <field name="amount_tax_company" optional="hide" readonly="1"/>
+                                    <field name="amount_tax_company" optional="hide" readonly="0"/>
                                     <field name="total_amount" options="{'currency_field': 'currency_id'}" string="Subtotal In Currency " optional="show" attrs="{'readonly': [('product_has_cost', '=', True)]}" groups="base.group_multi_currency"/>
-                                    <field name="total_amount_company" string="Subtotal" readonly="1"/>
+                                    <field name="total_amount_company" string="Subtotal" readonly="0"/>
```

Comment:
just remove the attribute

Reply:
But would it let it on readonly if you upgrade though? (don't remember if I tried and had to override it to 0)